src/ocamlorg_frontend/pages/outreachy.eml

```diff
@@ -0,0 +1,56 @@
+let render (metadata : Ood.Outreachy.t list)
+= 
+Layout.render
+~title:"OCaml Outreachy Internships"
+~description:"Records Of OCaml Community Past Outreachy Internship Projects"
+~canonical:Url.outreachy @@
+<div class="intro-section-simple">
+    <div class="container-fluid">
+        <div class="text-center prose w-full lg:w-2/3 m-auto">
+            <h1 class="font-bold mb-6">OCaml Outreachy Internships</h1>
+            <p>This is record of all past OCaml Community <a href="https://www.outreachy.org/" target="_blank">Outreachy</a> Internship Projects.</p>
+        </div>
+    </div>
+</div>
+
+<div class="py-12 bg-white">
+  <div class="container relative mx-auto px-4 items-center">
+          <% metadata |> List.iter (fun (rounds : Ood.Outreachy.t) -> %>
+              <div class="block border-b border-[#EF8316] p-6 text-left my-2">
+                  <h2 class="font-bold"><%s rounds.name %></h2>
+              </div>
+              <div class="grid grid-cols-1 auto-rows-max gap-10">
+                  <% rounds.projects |> List.iter (fun (project : Ood.Outreachy.project) -> %>
+                          <div x-data="{'is_shown': false}" class="p-5 rounded-2xl items-center w-full max-w-prose">
+                              <div class="flex flex-col h-full space-y-2">                 
```

Comment:
```suggestion
                          <div x-data="{'is_shown': false}" class="flex flex-col gap-2 p-5 max-w-prose">
```

These two `div`s can be collapsed into a single div and useless classes can be removed.

Reply:
it breaks the layout when I do that

Reply:
You may need to adjust the classes. For example, you could try to move the `max-w-prose` class to the parent div, so that the parent div already restricts the width.